service/github/check.go

```diff
@@ -78,9 +78,14 @@ func (ch *Check) GetResult() *CheckResult {
 	return ch.result
 }
 
+func (ch *Check) SetToolName(toolName string) {
```

Comment:
**[golint-circleci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>exported method Check.SetToolName should have comment or be unexported
<!-- __reviewdog__:ChBlZGE5YWE4NTk5NmU5ODQ5Eg9nb2xpbnQtY2lyY2xlY2k= -->